### ISSUE TITLE
Material datepicker error - Cannot read properties of null (reading 'required')

### ISSUE THREAD

**Issue by VenkataPakalapati** [ngx-formly/ngx-formly#3315]
**Description**
upgraded formly to 5.12.2 and the datepicker is broken

```
TypeError: Cannot read properties of null (reading 'required')
    at ngx-formly-material-datepicker.js:71:34
    at wrapProperty (ngx-formly-core.js:327:9)
    at FormlyFieldDatepicker.ngAfterViewInit (ngx-formly-material-datepicker.js:66:26)
    at callHook (core.mjs:2551:22)
    at callHooks (core.mjs:2520:17)
    at executeInitAndCheckHooks (core.mjs:2471:9)
    at refreshView (core.mjs:9566:21)
    at refreshEmbeddedViews (core.mjs:10646:17)
    at refreshView (core.mjs:9519:9)
    at refreshComponent (core.mjs:10692:13)
```

**Environment**

- Angular version: 13.3.8
- Formly version: 5.12.2

**Additional context**

looks like there is no null check for **currentValue** here

<img width="558" alt="Screen Shot 2022-05-26 at 7 32 15 PM" src="https://user-images.githubusercontent.com/17863587/170701337-3397463c-71a4-4f14-bc5f-efbbc81aa64c.png">

we were able to get around this for now by creating a clone of datepicker type and adding a null check for **currentValue**.

**Comment by VenkataPakalapati** [comment]
Thanks @aitboudad for the quick turnaround. When can we expect this fix to be available in R6?

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.12.3](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.12.3) and [v6.0.0-beta.2](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-beta.2) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by aitboudad** [comment]
> When can we expect this fix to be available in V6?

yes
